src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderWidget/PaymentBuilderWidget.tsx

```diff
@@ -91,7 +101,12 @@ const PaymentBuilderWidget: FC<PaymentBuilderWidgetProps> = ({ action }) => {
     {
       key: ExpenditureStep.Create,
       heading: { label: formatText({ id: 'expenditure.createStage.label' }) },
-      content: (
+      content: isStaked ? (
+        <FinalizeWithStakingInfo
```

Comment:
I believe this component (and related ones) originally displayed motion/expenditure finalization info but it's now used in more places such as creation in this case. Could the naming reflect that better as technically you cannot finalize with staking?

Reply:
Will `StepFinalizedWithStakingInfo` be a better name? If not, maybe you can suggest something?

Reply:
@joanna-pagepro sorry missed your comment, there's no finalization with staking, in this case it's the create step. `CreatedWithStakingInfo` or similar would make sense for this scenario (it's the only action you can do with staking). In other places where `FinalizeWithPermissionsInfo` might be a bit confusing (it's used in cancel, create, review and release steps), maybe something generic such as `ActionWithPermissionsInfo`?